include/thread_pool/mpmc_bounded_queue.hpp

```diff
@@ -76,19 +76,40 @@ class MPMCBoundedQueue
     MPMCBoundedQueue& operator=(MPMCBoundedQueue&& rhs) noexcept;
```

Comment:
This class obviously should not have a copy constructor and copy assignment operator. It's a good practice to explicitly delete them.

Reply:
Couldn't this still be useful if ulterior synchronization is used to ensure `rhs` has entered a defined state?

Reply:
Fixed in #3.